src/AccessibilityInsights.SharedUx/Controls/TestTabs/TabStopControl.xaml

```diff
@@ -45,8 +45,8 @@
             <TextBlock Grid.Row="2" Grid.Column="1" Grid.ColumnSpan="2" VerticalAlignment="Center" Margin="0,0,0,10">
                 <Run FontWeight="SemiBold" FontSize="{DynamicResource ConstXLTextSize}" Text="{x:Static Properties:Resources.RunTextTabStops}" />
                 <Hyperlink TextDecorations="{x:Null}" FocusVisualStyle="{DynamicResource {x:Static SystemParameters.FocusVisualStyleKey}}"
-                           ToolTip="{x:Static Properties:Resources.RunTextTabStops}"
-                           AutomationProperties.Name="{x:Static Properties:Resources.RunTextHowToTest}"
+                           ToolTip="{x:Static Properties:Resources.RunTextGuidance}"
+                           AutomationProperties.Name="{x:Static Properties:Resources.RunTextGuidance}" AutomationProperties.HelpText=" "
```

Comment:
HelpText is set here to stop it from being automatically set the the ToolTip text